packages-exp/firebase-exp/gulpfile.js

```diff
@@ -22,7 +22,10 @@ var sourcemaps = require('gulp-sourcemaps');
 const OUTPUT_FILE = 'firebase.js';
 const pkgJson = require('./package.json');
 const files = [
-  ...pkgJson.components.map(component => `firebase-${component}.js`)
+  ...pkgJson.components.map(component => {
```

Comment:
I guess it was there before and it's not very important but [...array] seems unnecessary when there's only one item, it could just be array